docker/build/docker.ubuntu.Dockerfile

```diff
@@ -1,10 +1,21 @@
 ARG ARCH="amd64"
 
 FROM ${ARCH}/ubuntu:18.04
-RUN apt-get update && apt-get install curl python build-essential apt-transport-https ca-certificates software-properties-common -y && \
+RUN apt-get update && apt-get install curl python python3.7 python3-pip build-essential apt-transport-https ca-certificates software-properties-common -y && \
     curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - && \
     add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" && \
     apt-get update && apt-get install docker-ce -y
+
+# Mule needs >= python3.7 so set that as the default.
```

Comment:
We shouldn't need to install mule in any of the dockerfiles we're maintaining in this repo

Reply:
Could you explain this more?  How would you suggest that we get mule?